README.md

```diff
@@ -56,3 +56,13 @@
 ### [How to create a Conan package for prebuilt binaries in a remote](tutorial/creating_packages/prebuilt_remote_binaries/)
 
 - Learn how to create a Conan package when we have prebuilt libraries located in a remote repository. [Docs](https://docs.conan.io/en/2.0-alpha/tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.html#downloading-and-packaging-pre-built-binaries)
+
+## Examples
+
+### [Use custom commands in your Conan CLI](examples/extensions/commands/)
+
+- Learn how to create custom commands in Conan. [Docs](https://docs.conan.io/en/2.0-alpha/reference/commands/custom_commands.html)
+
+#### [clean:revisions command](examples/extensions/commands/clean/cmd_revisions.py)
+
+- ``conan clean:revisions`` command: Deletes (from local cache or remotes) all recipe and package revisions but the latest package revision from the latest recipe revision. [Docs](https://docs.conan.io/en/2.0-alpha/examples/extensions/commands/clean/custom_command_clean_revisions.html)
```

Comment:
This should be just one entry to the examples, you can use a subtitle to group the custom commands, but I think there's no need now as we have just one